README.md

```diff
@@ -131,15 +153,15 @@ Copy and paste `IBAnimatable` folder in your Xcode project.
 
 ### [Swift package manager](https://swift.org/package-manager)
 
-Add `.Package(url: "https://github.com/IBanimatable/IBanimatable.git", majorVersion: 2)` to your `Package.swift`
+Add `.Package(url: "https://github.com/IBanimatable/IBanimatable.git", majorVersion: 3)` to your `Package.swift`
 
 ### [CocoaPods](https://cocoapods.org)
 
 Add `pod 'IBAnimatable'` to your Podfile.
 
 ### [Carthage](https://github.com/Carthage/Carthage)
 
-Add `github "IBanimatable/IBAnimatable" ~> 2` to your Cartfile.
+Add `github "IBanimatable/IBAnimatable" ~> 3` to your Cartfile.
```

Comment:
You can just put `github "IBanimatable/IBAnimatable"` in your Cartfile, it's not mandatory to specify the version in the README. That will make each new release easier for you